src/app/base/components/record-card/components/record-display/record-display.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { RecordCardOptions } from '../../record-card-helper';
+import { RecordDisplay } from '../../record-card.types';
+
+@Component({
+  selector: 'kern-record-display',
+  templateUrl: './record-display.component.html',
+  styleUrls: ['./record-display.component.scss']
+})
+export class RecordDisplayComponent implements OnChanges {
+
+  @Input() record: RecordDisplay;
+  @Input() attributes: any;
+  @Input() recordCardOptions: RecordCardOptions;
+
+  constructor() { }
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.record) {
+      if (!this.record.hasOwnProperty('data')) {
+        if (this.record.hasOwnProperty('fullRecordData')) {
+          this.record.data = this.record.fullRecordData;
+        }
+        if (this.record.hasOwnProperty('recordData')) {
```

Comment:
shouldn't this be else if? or is this intentionally overwritten when both exist?

Reply:
also an else if neither exists with an error to ensure the developer uses it right

Reply:
Yes it should be else if (I think either recordData or fullRecordData can be replacement for data but not both of them at the same time)